Makefile

```diff
@@ -23,3 +24,5 @@ test:
 setup:
 	virtualenv venv
 	cp wikipendium/settings/local.py.example wikipendium/settings/local.py
+
+.PHONY: all run serve update migrate migrations lint test setup
```

Comment:
I prefer the latter, since it makes it easier to remember to update the phonies.